src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -30,7 +30,7 @@ public class ModelManager implements Model {
     private final AddressBook addressBook;
     private final UserPrefs userPrefs;
     private final FilteredList<Client> filteredClients;
-    private final SortedList<Client> sortedClients;
+    private final SortedList<Client> sortedFilteredClients;
```

Comment:
it may have a issue for filteredClients and sortedFilteredClients at the same time.

Reply:
filteredClients will not be accessed by the UI or other components, only sortedFilteredList will be displayed